api/index.js

```diff
@@ -12,56 +12,46 @@ var errors = require('./lib/errors');
 var serverLib = require('./lib/server-lib');
 var remote = require('./lib/remote');
 
-module.exports = {
-  info: {
-    serverStatus: Info.serverStatus,
-    isConnected: Info.isConnected,
-    fee: Info.fee,
-    uuid: Info.uuid
-  },
+function RippleAPI() {
+  this.remote = remote;
+}
```

Comment:
`this` is a class instance scope, not really a global - it's just that the class methods are split over multiple files. The way you have it written won't work because each instance will have it's own remote object in the next pull request. We could do something like this in index.js:

    var Info = require('./info');
    ...
    RippleAPI.prototype.getFee = function() {
        return Info(this.remote).getFee.apply(this, arguments);
    }
    RippleAPI.prototype.getUUID = function() {
        return Info(this.remote).getUUID.apply(this, arguments);
    }
    ...

and in info.js:

    var curry = require('curry');
    ...
    module.exports = function(remote) {
        return {
            getFee: curry(getFee)(remote),
            getUUID: curry(getUUID)(remote),
            ...
        };
    }

then add a `remote` parameter to every method.

Another option is to not change `module.exports`, but still add the `remote` parameter to every method and then in index.js do:

    var Info = require('./info');
    var curry = require('curry');
    ...
    RippleAPI.prototype.getFee = function() {
        return curry(Info.getFee)(this.remote).apply(this.arguments);
    }

I'm not really sure if either of these options is more clear than the current form.